packages/gestalt/src/Button.js

```diff
@@ -152,9 +120,12 @@ const ButtonWithForwardRef: AbstractComponent<unionProps, unionRefs> = forwardRe
     selected = false,
     size = 'md',
     text,
-  } = props;
-
-  const innerRef = useRef<null | HTMLAnchorElement | HTMLButtonElement>(null);
+    type,
+    ...props
```

Comment:
can we destructure it all? it's just name no? now both use name

Reply:
DONE